parcels/tools/exampledata_utils.py

```diff
@@ -1,16 +1,33 @@
 import os
 from datetime import datetime, timedelta
 from pathlib import Path
-from urllib.request import urlretrieve
 
-import platformdirs
+import pooch
 import xarray as xr
 
 from parcels.tools._v3to4 import patch_dataset_v4_compat
 
-__all__ = ["download_example_dataset", "get_data_home", "list_example_datasets"]
-
-example_data_files = {
+__all__ = ["download_example_dataset", "list_example_datasets"]
+
+# When modifying existing datasets in a backwards incompatible way,
+# make a new release in the repo and update the DATA_REPO_TAG to the new tag
+DATA_REPO_TAG = "main"
+
+DATA_URL = f"https://github.com/OceanParcels/parcels-data/raw/{DATA_REPO_TAG}/data"
+
+# Keys are the dataset names. Values are the filenames in the dataset folder. Note that
+# you can specify subfolders in the dataset folder putting slashes in the filename list.
+# e.g.,
+# "my_dataset": ["file0.nc", "folder1/file1.nc", "folder2/file2.nc"]
+# my_dataset/
+# ├── file0.nc
+# ├── folder1/
+# │   └── file1.nc
+# └── folder2/
+#     └── file2.nc
+#
+# See instructions at https://github.com/OceanParcels/parcels-data for adding new datasets
```

Comment:
@fluidnumerics-joe are these instructions clear? Do you think you can add your datasets now?